spec/factories/offenders.rb

```diff
@@ -0,0 +1,14 @@
+FactoryBot.define do
+  factory :offender, class: 'Nomis::Offender' do
+    inprisonment_status do 'SENT03' end
+    sentence do
+      Nomis::SentenceDetail.new.tap { |s|
```

Comment:
I note that this `.tap` multiline block uses `{...}` curly braces style, although I've seen this same style used elsewhere specifically for `.tap` blocks.

It looks like both our rubocop configs (the main `.rubocop.yml` and the aspirational `.rubocop-govuk.yml`) don't seem to have an opinion on `do...end` vs `{...}` – so maybe this is a non-issue.